tests/run/value-class-extractor-seq.scala

```diff
@@ -1,8 +1,12 @@
 import scala.runtime.ScalaRunTime.stringOf
 
 final class ArrayOpt[T](val xs: Array[T]) extends AnyVal {
+  def toSeq: Seq[T] = xs.toSeq
+  def length: Int   = xs.length
   def isEmpty = xs == null
-  def get = xs
```

Comment:
I'm guessing this change is needed because `Array[T]` only defines `length` `apply` and `drop` after implicit conversion to `ArrayWrapper`, which dotty doesn't perform in that case...

Reply:
Yes, the new Spec requires `toSeq` as well.